Ginger/GingerCoreCommonTest/VariableTests/SelectionListVariableTests.cs

```diff
@@ -90,7 +90,7 @@ public void SelectionListVar_TestImageType()
             eImageType eImageType = variableSelectionList.Image;
 
             //Assert
-            Assert.AreEqual(eImageType.VariableList, eImageType, "Image Type");
+            Assert.AreEqual(eImageType.MinusSquare, eImageType, "Image Type");
```

Comment:
why you chaging the image types Prateek was setting?

Reply:
this testcases were failing so when debugged then found it is not checking correctly so changed the type in testcase

Reply:
but now Asserts are not correct, please show it to Prateek and make the unit tests work as expected with correct Asserts